core/src/main/java/org/helsinki/paytrail/model/payments/PaytrailPaymentResponse.java

```diff
@@ -1,22 +1,23 @@
 package org.helsinki.paytrail.model.payments;
 
+import com.fasterxml.jackson.databind.JsonNode;
 import lombok.Data;
+import lombok.EqualsAndHashCode;
 import lombok.experimental.Accessors;
+import org.helsinki.paytrail.response.PaytrailResponse;
 
 import java.io.Serializable;
 import java.util.List;
 
-import com.fasterxml.jackson.databind.JsonNode;
-
 @Data
+@EqualsAndHashCode(callSuper = true)
 @Accessors(chain = true)
-public class PaytrailPaymentResponse implements Serializable {
+public class PaytrailPaymentResponse extends PaytrailResponse implements Serializable {
```

Comment:
This inheritance is unnecessary, because PaytrailPaymentCreateResponse already extends it. All the fields in PaytrailResponse is available from there.

Reply:
Fixed